CHANGELOG.md

```diff
@@ -41,6 +43,7 @@
   - `regr_sxx`
   - `regr_sxy`
   - `regr_syy`
+  - `array_slice`
   - `try_to_binary`
```

Comment:
did this change get removed?

Reply:
+ need to update functions.rst?

Reply:
I found out that we already support it, so I removed it from this PR